docs/docs/30-administration/22-backends/20-ssh.md

```diff
@@ -0,0 +1,11 @@
+# SSH backend
+
+The local backend will execute the pipelines using SSH on a remote system without any isolation of any kind.
+
+Since the code run directly in the same context as the agent (same user, same filesystem), a malicious pipeline could 
```

Comment:
However, it doesn't run in the same context.

Reply:
I should have looked at the docs after I copied them before commiting 😆